### ISSUE TITLE
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color, at Dashboard screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#857]
**Describe the bug**
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color.

**To Reproduce**
1. Install "Color Contrast Analyzer" extension in your browser.
2. Go to Dashboard screen "http://localhost:9992/cm/app/?view=dash"
3. Click "Extension" icon given in the chrome browser, in top right. It will open Extension Pop Up.
4. Click "Color Contrast Analyzer", option in pop up. It will open another pop up.
5. Click "Capture Visible Content" from the pop up.
6. Observe that the folder image present before "Assets", "Design" etc is not highlighted by the analyzer. So, that means they don't have the desired contrast ratio.

**Expected behavior**
The visual presentation of the Components/Images should have a contrast ratio of at least 3:1 against adjacent color.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by natechadwick** [comment]
@rgarg24 Why so long on this one, isn't it just recoloring a few images?

**Comment by rgarg24** [comment]
@natechadwick This one requires checking for recoloring of images for all the dashboard gadgets. That's why it will take time.
